cmd/net.go

```diff
@@ -66,18 +67,37 @@ func mustGetLocalIP4() (ipList set.StringSet) {
 // getHostIP4 returns IPv4 address of given host.
 func getHostIP4(host string) (ipList set.StringSet, err error) {
 	ipList = set.NewStringSet()
-	ips, err := net.LookupIP(host)
-	if err != nil {
-		return ipList, err
-	}
-
-	for _, ip := range ips {
-		if ip.To4() != nil {
-			ipList.Add(ip.String())
+	//variable to list resolved IP addresses
+	var ips []net.IP
+	// channel to indicate completion of host resolution
+	doneCh := make(chan struct{})
+	// Indicate to our retry routine to exit cleanly, upon this function return.
+	defer close(doneCh)
+	// Wait on the jitter retry loop
+	retryTimeCh := newRetryTimerSimple(doneCh)
+	for {
+		select {
+		case <-retryTimeCh:
+			// try to resolve remote host
+			ips, err = net.LookupIP(host)
+			if err != nil {
+				// log error and continue. We dont fail on host resolution, just log
+				// the message to console about the host not being resolveable. It is
+				// expected that user provides resolveable hosts
+				errorIf(err, "Unable to resolve host %s (elapsed %s)", host, getElapsedTime(time.Now().Round(time.Second)))
```

Comment:
1. I think it is important to convey to the user how long Minio server has been waiting to resolve the host. This will help to user to make a decision whether it is okay to wait further or there is some issue with the remote hosts they provided to Minio

2. How is it incorrect?